extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -110,3 +110,92 @@
     opacity: .4;
   }
 }
+
+/* Disclaimer CSS rules */
+.i-amphtml-story-interactive-disclaimer {
+  position: absolute !important;
+  color: black !important;
+  bottom: calc(100% - 1.75em) !important;
+  right: 0 !important;
+  z-index: 1 !important;
+}
+
+.i-amphtml-story-interactive-container:not(.i-amphtml-story-interactive-active) .i-amphtml-story-interactive-disclaimer {
+  display: none !important;  /* Disable the disclaimer if not on active page to prevent tabbing */
+}
+
+.i-amphtml-story-interactive-disclaimer-alert {
+  width: 1.75em !important;
+  height: 1.75em !important;
+  position: absolute !important;
+  right: 0 !important;
+  bottom: 0 !important;
+  transform-origin: bottom right !important;
+  pointer-events: all !important;
+  background-color: var(--interactive-accent-color) !important;
+  border: none !important;
+  border-radius: 50% !important;
+  box-shadow: 0 .2em .2em 0 #00000040 !important;
+  padding: .25em !important;
+  cursor: pointer !important;
+  font-size: inherit !important;
+}
+
+.i-amphtml-story-interactive-disclaimer[active]
+  .i-amphtml-story-interactive-disclaimer-alert {
+  display: none !important;
```

Comment:
This causes a slight flash on navigation since `CURRENT_PAGE_ID` service is slightly async.
![Dec-17-2020 11-10-26](https://user-images.githubusercontent.com/3860311/102517178-bce20f80-405d-11eb-840e-dbb2d875115a.gif)

We could set tabindex within the `CURRENT_PAGE_ID` listener.

It's not perfect though because we might want to set a tabindex on the options buttons when we work on a11y for that.

This is a tricky one since aria-hidden doesn't stop tabbing to a focusable element.
https://web.dev/aria-hidden-focus/

Reply:
Setting the tabindex is much more involved (because we have two buttons and a link that we'd need to change the tabindex) versus just hiding it, so there's a tradeoff. Given that the flash only happens because those two elements are exactly the same, I'd assume that in any other case where the alert doesn't show exactly on the same spot it wouldn't be seen as a flash.

However, we will have to take care of the options when we make them buttons... WDYT if we do `this.element.querySelectorAll("a, button").forEach((e) => e.setAttribute("tabindex", "-1"))` (and `setAttribute("tabindex", "0")` to enable)?

Reply:
The flash is visible when the elements are in different positions.
![Dec-17-2020 12-25-27](https://user-images.githubusercontent.com/3860311/102521530-0da83700-4063-11eb-8b21-34824c10ee8c.gif)

I think that sounds good for the tabindex. If we want to set an order of tabindex in the future we would have to change this but maybe this works for now.